code/game/objects/obj_defense.dm

```diff
@@ -72,7 +72,7 @@
 		return FALSE
 	if(QDELETED(src))
 		return FALSE
-	fire_act()
+	fire_act(60)
```

Comment:
Probably should be a define, especially since it is used twice

Reply:
done